lib/data_update_scripts/20210203104631_add_single_resource_admin_role_to_users_with_tech_admin.rb

```diff
@@ -0,0 +1,10 @@
+module DataUpdateScripts
+  class AddSingleResourceAdminRoleToUsersWithTechAdmin
+    def run
```

Comment:
Wanted a second pair of 👀's on this to double check the performance of this query. Maybe there's a way to improve it (especially since we're working with the users).

Reply:
I think the script is fine. there's still one thing left: how do we make sure that each time a user is given the `tech_admin` role they are also given the single resource admin? Otherwise this will only work for existing tech admins, not future ones

Reply:
@rhymes what do you think of the ffg:

We can add the single resource admin (DataScript) role to a user when the admin chooses to give a user a tech_admin role under Manage Users. Since this seems to be the only place to assign user roles (from my brief  investigation) it would be a safe solution. This can be done in ManageActivityAndRoles.

However, it seems that tech_admin up until now has not even been assigned through the UI (based on the fact that it's not in the dropdown):
<img width="483" alt="Screenshot 2021-02-04 at 13 40 54" src="https://user-images.githubusercontent.com/2786819/106889149-4a51ef80-66f0-11eb-8c06-19572eb1ea26.png">.
I'd have to add it to the dropdown first of course.

It does make me a bit uneasy to be adding code along to the tech_admin when it seems like more decisions need to be made about how this role behaves.

Reply:
✅ Done